microsoft/testsuites/libvirt/libvirt_tck.py

```diff
@@ -28,6 +30,15 @@
     """,
 )
 class LibvirtTckSuite(TestSuite):
+    def before_case(self, log: Logger, **kwargs: Any) -> None:
+        node = kwargs["node"]
+        if not isinstance(node.os, Ubuntu):
```

Comment:
Is it a hard requirement of the libvirt tests, or just this test case doesn't support other distros?

Reply:
Not a requirement of the libvirt tests AFAIK. Just not implemented in this test case.